src/detection/cpu/cpu_linux.c

```diff
@@ -247,6 +248,9 @@ static const char* parseCpuInfo(
             #if !(__arm__ || __aarch64__)
             (cpu->name.length == 0 && ffParsePropLine(line, "model name :", &cpu->name)) ||
             (cpu->vendor.length == 0 && ffParsePropLine(line, "vendor_id :", &cpu->vendor)) ||
+            //Is it cheaper to just parse every physical id or to check if it's already set to the parsed value?
```

Comment:
I am not sure whether it's the most optimal way to go about things. The intended logic is to see how many different physical ids are present in cpuinfo.

Reply:
I guess we just need to fix the maximum id?

Reply:
I don't think this code works. Currently [the loop exits after the first core entry](https://github.com/fastfetch-cli/fastfetch/pull/1417/files#diff-6f3a2f89998ef2269032903b3586e2c86b1f0720760190e4785e7db3a0c6a72fR237-R243), while `physical id` requires enumerating every entry.